WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/ShopinBit/1 - Initial/StartConversationStep.cs

```diff
@@ -55,7 +55,7 @@ private async Task<Conversation> StartNewConversationAsync(Conversation conversa
 			_ => "a few days"
 		};
 
-		conversation = conversation.AddBotMessage($"Thank you! We've received your request and will get in touch with you within {hourRange} (Monday to Friday).");
+		conversation = conversation.AddBotMessage($"Thank you! We've received your request and we'll get in touch with you within {hourRange} (Monday to Friday).");
```

Comment:
@molnard If for some reason you don't want to include this PR, please at least include this